dask/backends.py

```diff
@@ -34,27 +40,27 @@ def detect_entrypoints(entry_point_name):
 BackendFuncType = TypeVar("BackendFuncType", bound=Callable[..., Any])
```

Comment:
This `BackendFuncType` definition can now be removed, correct?

Reply:
That's right, good catch.